projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -368,7 +368,6 @@ export class DataTableBodyComponent<TRow extends Row = any> implements OnInit, O
   @Output() page = new EventEmitter<number>();
   @Output() activate = new EventEmitter<ActivateEvent<TRow>>();
   @Output() select = new EventEmitter<SelectEvent<TRow>>();
-  @Output() detailToggle = new EventEmitter<any>();
```

Comment:
This is actually a user-exposed event, not an internal one. It's also referenced in the documentation, see https://github.com/siemens/ngx-datatable/blob/3f0c2fe9f73b349213272fef71ee035c31f40d0e/docs/api/table/outputs.md?plain=1#L21.

So I'm not entirely sure if it is a good idea to get rid of it since people might still rely on it. If we do want to get rid of it, we should also purge all the other references: https://github.com/search?q=repo%3Asiemens%2Fngx-datatable%20detailToggle&type=code.

Reply:
Should I silently drop this from the outputs.md?

Reply:
@spike-rabbit as discussed offline: While the `detailToggle` event seems to have been inteded to be an exposed one on the top-level component, it's currently not and hence only internal, unlike what the documentation implies. This is not only the case for our fork but also the upstrea project. Since it is already possible to subscribe to that even over other means, it's indeed not a necesary event and the lack of implementing the output emphasizes that. 

Therefore, let's prune the event from all locations, like mentioned docs (not sure if there are any other ones). Could you please do so? Many thanks in advance 🙇

Reply:
Output.md is adjusted.